gui/projectfiledialog.cpp

```diff
@@ -482,8 +489,10 @@ void ProjectFileDialog::saveToProjectFile(ProjectFile *projectFile) const
         codingStandards << CODING_STANDARD_CERT_CPP;
     if (mPremium && mUI->mMisraVersion->currentIndex() == 1)
         codingStandards << CODING_STANDARD_MISRA_C_2023;
-    if (mUI->mMisraCpp2008->isChecked())
+    if (mPremium && mUI->mMisraCppVersion->currentIndex() == 0)
```

Comment:
you should check that mMisraCpp is checked also as far as I know the currentIndex can be 0 even though the combobox is disabled.